pkg/skaffold/plugin/environments/gcb/desc.go

```diff
@@ -50,6 +53,8 @@ func (b *Builder) buildDescription(artifact *latest.Artifact, tag, bucket, objec
 
 func (b *Builder) buildSteps(artifact *latest.Artifact, tag string) ([]*cloudbuild.BuildStep, error) {
 	switch {
+	case artifact.BuilderPlugin != nil:
```

Comment:
I am guessing you can't have multiple values say BuilderPlugin and ArtifactType  set for the same Artifact? Is that true

Reply:
yes, that's true!